package.json

```diff
@@ -140,6 +140,7 @@
     "react-native-crypto": "2.2.0",
     "react-native-device-info": "6.0.1",
     "react-native-easy-markdown": "1.4.1",
+    "react-native-easy-view-transformer": "^1.2.0",
```

Comment:
no updates in 3 years
have you looked into alternatives?

Reply:
I've looked into multiple libraries, and this seems to be the best one. All libs usually have 3-4 years with no update. https://www.npmjs.com/package/react-native-image-zoom-viewer  https://www.npmjs.com/package/react-native-photo-zoom the "maintained" ones have very few downloads. The one I picked works fine and has a single purpose. We can revise later if there are some issues. What are your thoughts?